spyder/plugins/editor.py

```diff
@@ -631,6 +632,15 @@ def get_plugin_actions(self):
         self.register_shortcut(self.new_action, context="Editor",
                                name="New file", add_sc_to_tip=True)
 
+        add_shortcut_to_tooltip(self.new_action, context="Editor",
+                                name="New file")

```

Comment:
Please remove this couple of lines, and the import of `add_shortcut_to_tooltip`. They are not needed anymore.